tools/cockpit.spec

```diff
@@ -471,7 +471,6 @@ authentication via sssd/FreeIPA.
 %doc %{_mandir}/man8/cockpit-ws.8.gz
 %doc %{_mandir}/man8/cockpit-tls.8.gz
 %doc %{_mandir}/man8/remotectl.8.gz
-%doc %{_mandir}/man8/pam_cockpit_cert.8.gz
 %doc %{_mandir}/man8/pam_ssh_add.8.gz
 %dir %{_sysconfdir}/cockpit
 %config(noreplace) %{_sysconfdir}/cockpit/ws-certs.d
```

Comment:
The `%config(noreplace) %{_sysconfdir}/pam.d/cockpit` in the next line bites a bit here. (See [docs](https://www.cl.cam.ac.uk/~jw35/docs/rpm_config.html)). It behaves similar to dpkg's conffiles. So at some point when we drop the PAM stub we need to apply some seddery in the `%post`/.postinst script to fix up the PAM file.

But for now I think we should add a grep to the post scripts that writes a deprecation warning if the PAM file still contains `pam_cockpit_cert`.